### ISSUE TITLE
Better resizability 2.0

### ISSUE THREAD

**Issue by mmahmoudian** [h3poteto/whalebird-desktop#2494]
## Describe
This is sort of a follow-up on #758 and #1046.

### The issue
At the moment with the latest release (Whalebird-4.4.1-linux-x64.AppImage) everything works fine with narrow width window (similar to what the official twitter app, Pidgin and other narrow-width app does). But the preference is still designed for wide window. At the moment the text overflows outside of the field of view:

![Peek 2021-06-03 13-05](https://user-images.githubusercontent.com/390889/120628395-51ee8e80-c46d-11eb-8adc-c94a076cdda5.gif)

### Proposal
Either pop out the preferences in a new window, or make the side-bar of the preferences window much smaller to accommodate the settings to fit into the page.

## Why
This would help user to properly use the preferences even when the window is narrow.

**Comment by h3poteto** [comment]
I see, I will consider the narrow design.

**Comment by mmahmoudian** [comment]
I dont know if this bot can be configured to avoid tagging feature requests. Afterall, a feature request can remain valid and yet stale for years. The activity mostly depends on the developers and contributors imho.

**Comment by h3poteto** [comment]
I'm not going to exclude stale bot, even if it's a feature request. Because if it hasn't been discussed for a long time and hasn't been implemented, I wonder if it's a really necessary.

**Comment by mmahmoudian** [comment]
@h3poteto your software, your repo, your rules. Do as you wish and as you see fit 👍

**Comment by xabirequejo** [comment]
(I know that this issue is about settings on narrow screens but I didn't want to open a new one for resizeability.)

- I have realised that the contextual menu (?) breaks words because they don't fit. It would be nice if the width of the menu was dynamic and would adapt to the length of the strings.
- Also, some of the strings are too long for the sidebar and they are shown cramped or cut because they don't fit. I have changed the localisation to use shorter strings as a quick fix.

I think it would be nice if we were able to drag the sidebar to make it as wide or as narrow as we wanted or for better readability. Below X pixels the words would be shown as icons, the same way it does now when you collapse the sidebar.

![whalebird-context-menu](https://user-images.githubusercontent.com/888924/227995599-96d887d1-531d-489a-9dea-5797650ec3f2.jpg)

![whalebird-sidebar](https://user-images.githubusercontent.com/888924/227984228-eebcb7e4-6a19-496f-b6f1-c48076b6dce0.jpg)
